application.go

```diff
@@ -92,6 +106,11 @@ func (a *Application) Run() error {
 	glfw.DefaultWindowHints()
 	defer a.window.Destroy()
 
+	a.resourceWindow, err = createResourceWindow(a.window)
+	if a.resourceWindow != nil {
```

Comment:
What if it is nil?

Reply:
Ah, I see it is handled below.. So we're being graceful here, if no window can be created, the app will work like it does currently..

Reply:
I think we should check the error above though. Not to exit, but just to print a message with the reason why no resource window could be created.

Reply:
Done!